src/call_log/call_log.c

```diff
@@ -217,4 +217,27 @@ __wt_call_log_open_session(WT_SESSION_IMPL *session, int ret_val)
     return (0);
 }
 
+/*
+ * __wt_call_log_set_timestamp --
+ *     Print the call log entry for the set timestamp API call.
+ */
+int
+__wt_call_log_set_timestamp(WT_SESSION_IMPL *session, const char *config, int ret_val)
+{
+    WT_CONNECTION_IMPL *conn;
+    char config_buf[128];
+    char objectid_buf[128];
```

Comment:
could we define them in one line? 
`char config_buf[128], objectid_buf[128];`

Reply:
Can do